### ISSUE TITLE
Can not run Example project - Shell Script Invocation Error

### ISSUE THREAD

**Issue by quachsimon** [blackuy/react-native-twilio-video-webrtc#270]
### Steps to reproduce
1. Pull in new project
2. Follow README 
3. Error

After I run it I run into this:

`
/Users/username/Library/Developer/Xcode/DerivedData/Example-cazqmojhumzmibgqvdtvyxngkwnd/Build/Intermediates.noindex/Example.build/Debug-iphonesimulator/Example.build/Script-00DD1BFF1BD5951E006B06BD.sh: line 3: ../node_modules/react-native/packager/react-native-xcode.sh: No such file or directory
`

Is anyone else running into this issue?

Trying to run on Xcode 11.3.1.

I want to try updating the Twilio version but can not run the project at all.

Thanks

**Comment by quachsimon** [comment]
After some digging, I was finally able to run the Example proj for iOS.

Two big errors that I ran into were:

1. The script execution error - 
`No such file or directory Command PhaseScriptExecution failed with a nonzero exit code /node_modules/react-native/packager/react-native-xcode.sh`

2. Module preset error - 
` Cannot find module 'babel-preset-react-native'`

I fixed both errors and was able to run the Example Proj.

Solutions:

1. Updated the script in `Example Proj > Build Phases > Bundle React Native Code and Images > 'packager' to 'scripts'`. Clean and re-run

2. Remove `.babelrc` file as per a newer version of RN (0.59.9 for this proj).

I've forked it out [here](https://github.com/blackuy/react-native-twilio-video-webrtc/compare/master...quachdev:Upgrade-Twilio-V3) if anyone wants to take a look at it (it's my personal account).

Ultimately I am going to try and update to the latest Twilio version (3) as I get more time to work on this. Will most likely need help on the Android side if anyone is more fluent in that realm.
